### PR TITLE
fix: made Snowplow tracker name, creation, and filtering configurable

### PR DESCRIPTION
**Pull request by justin-iteratively**
Hoping this will work for all use cases and preserve the current default.

```
// Default (current)
// * Only create a new tracker if one doesn't already exist
// * Events are sent to all Snowplow trackers
{
  forceNewTracker: false,
  newTrackerName: 'itly',
  trackerNameFilter: undefined,
}

// Dedicated tracker
// * Force creation of a dedicated tracker 'itly'
// * Events are only sent to the 'itly' tracker
{
  forceNewTracker: true,
  newTrackerName: 'itly',
  trackerNameFilter: 'itly',
}

// Track to multiple trackers
// * Force creation of a new tracker 'itly'
// * Events can be sent to multiple trackers if desired
{
  forceNewTracker: true,
  newTrackerName: 'itly',
  trackerNameFilter: 'itly;tracker2;',
}
```

### CODE REVIEW

**Comment by Qwal** [comment]
LGTM

**Review by akramarev — APPROVED** [review]
looks good to me, new option field names look a bit wordy, but I can't think of anything better.. Something like "createNewTracker", "trackerName", "trackerFilter" leave too much space for additional questions and require to check the docs.

**Comment by justin-iteratively** [comment]
@akramarev @ondrej @Qwal I reduced the options to be more concise. 

Reverted to creating a new tracker named 'itly' only if Snowplow is not already loaded.  The user can now specify what trackers to send events to via options.trackerNames.

**Review by ondrej — APPROVED** [review]
